### PR TITLE
Fixing around login()

### PR DESCRIPTION
**Pull request by mfalesni**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a8726cd9bfd3db92940b79fc6766e86e43daa148:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/login.py [hunk 1]
```diff
@@ -72,6 +71,12 @@ def log_in(self, user, method='click_on_login'):
             'password': user.credential.secret,
         })
         self.submit_login(method)
+        logged_in_view = self.browser.create_view(BaseLoggedInPage)
+        if logged_in_view.logged_in:
+            name = logged_in_view.current_fullname
+            if name != user.name:
+                self.logger.info('setting the appliance.user.name to %r', name)
```
Comment (psav) [inline cfme/login.py]: Isn't this dangerous, what if this gets called and we are logged in as a different person?

Comment (mfalesni) [inline cfme/login.py]: Unless we require that the User object passed via login must have name, what else can we do ... I can put hte none comparison instead

**Comment by dajoRH** [comment]
Lint report for commit 64b7073827988c9e5a8d864d462b52f87bfb844e:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**